### PR TITLE
fix(debian): Add php7.2 dependencies for Ubuntu Bionic

### PR DESCRIPTION
**Pull request by GMishx**
## Description

Added missing php7.2 dependencies for Ubuntu Bionic.

## How to test

Build debian packages in Ubuntu Bionic (18.04) and install them.

`dpkg-buildpackage -I`

### CODE REVIEW

**Comment by mcjaeger** [comment]
unfortunately something is wrong when building packages with json libs for c++:
```
make libfossologyCPP.a
make[5]: Leaving directory '/home/fossy/fossology/src/lib/cpp'
Package jsoncpp was not found in the pkg-config search path.
Perhaps you should add the directory containing `jsoncpp.pc'
to the PKG_CONFIG_PATH environment variable
No package 'jsoncpp' found
copyright.cc:56:10: fatal error: json/json.h: No such file or directory
 #include <json/json.h>
          ^~~~~~~~~~~~~
compilation terminated.
Makefile:70: recipe for target 'copyright.o' failed
make[4]: *** [copyright.o] Error 1
make[4]: Leaving directory '/home/fossy/fossology/src/copyright/agent'
Makefile:20: recipe for target 'all' failed
make[3]: *** [all] Error 2
make[3]: Leaving directory '/home/fossy/fossology/src/copyright'
Makefile:51: recipe for target 'build-copyright' failed
make[2]: *** [build-copyright] Error 2
make[2]: Leaving directory '/home/fossy/fossology/src'
Makefile:28: recipe for target 'build-src' failed
make[1]: *** [build-src] Error 2
make[1]: Leaving directory '/home/fossy/fossology'
debian/rules:35: recipe for target 'build-arch-stamp' failed
make: *** [build-arch-stamp] Error 2
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2
fossy@ubuntu:~/fossology$

```

**Comment by mcjaeger** [comment]
also composer could be installed:
```
make[1]: Leaving directory '/home/fossy/fossology/src/maintagent'
/usr/bin/make DESTDIR=/home/fossy/fossology/debian/fossology-common \
           PREFIX=/usr SYSCONFDIR=/etc/fossology LOCALSTATEDIR=/var \
           -C src composer_install
make[1]: Entering directory '/home/fossy/fossology/src'
current dir: '/home/fossy/fossology/src'
/usr/bin/install -D -m 644 composer.json composer.lock /home/fossy/fossology/debian/fossology-common/usr/share/fossology
(cd /home/fossy/fossology/debian/fossology-common/usr/share/fossology; export COMPOSER_HOME=/dev/null; composer install --no-plugins --no-scripts --no-dev)
/bin/sh: 1: composer: not found
Makefile:85: recipe for target 'composer_install' failed
make[1]: *** [composer_install] Error 127
make[1]: Leaving directory '/home/fossy/fossology/src'
debian/rules:71: recipe for target 'install-arch' failed
make: *** [install-arch] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
```

**Comment by mcjaeger** [comment]
and then ext-mbstring and ext-xml: 
```
  Problem 1
    - The requested PHP extension ext-mbstring * is missing from your system. Install or enable PHP's mbstring extension.
  Problem 2
    - The requested PHP extension ext-xml * is missing from your system. Install or enable PHP's xml extension.
...
```

**Comment by GMishx** [comment]
> unfortunately something is wrong when building packages with json libs for c++:

Should be fixed by #1242

**Comment by GMishx** [comment]
> also composer could be installed:

Should be fixed now.

> and then php-mbstring and php-xml:

I have `php-mbstring` to `fo-installdeps` but `php-xml` is already there. I can't understand why it is not working.

**Comment by mcjaeger** [comment]
OK, let me retest

File: utils/fo-installdeps [hunk 1]
```diff
@@ -167,7 +167,7 @@ if [[ $RUNTIME ]]; then
             stretch|buster|sid)
                apt-get $YesOpt install postgresql-9.6 php7.0 php7.0-pgsql php7.0-cli php7.0-curl php7.0-xml php7.0-zip s-nail libboost-program-options1.62.0 libboost-regex1.62.0;;
             bionic)
-               apt-get $YesOpt install postgresql-10 php7.2 php7.2-pgsql php7.2-cli php7.2-curl php7.2-xml php7.2-zip s-nail libboost-program-options1.65.1 libboost-regex1.65.1;;
+               apt-get $YesOpt install postgresql-10 php7.2 php7.2-pgsql php7.2-cli php7.2-curl php7.2-xml php7.2-zip php7.2-mbstring s-nail libboost-program-options1.65.1 libboost-regex1.65.1;;
```
Comment (mcjaeger) [inline utils/fo-installdeps]: I think this belongs in the debian control file, because php-mbstring is missing for older versions too.

Comment (mcjaeger) [reply utils/fo-installdeps]: also php-xml does not seem to be covered, because it is there. for php7.2 ... but,  I have reported both at the same time?

Comment (GMishx) [reply utils/fo-installdeps]: `php-mbstring` and `php-xml` are not part of Trusty and Jessie packages. Therefore they can not be put in the control file.

**Pull request merged**